README.md

```diff
@@ -69,11 +69,12 @@ lock(resource: 'staging-server', inversePrecedence: true) {
 }
 ```
 
-#### Resolve a variable configured with the resource name
+#### Resolve a variable configured with the resource name and properties 
 
 ```groovy
 lock(label: 'some_resource', variable: 'LOCKED_RESOURCE') {
   echo env.LOCKED_RESOURCE
+  echo env.LOCKED_RESOURCE_PROP_ABC
```

Comment:
@MarkEWaite do you know, how can be this hidden in plugin page until new version has been released?  When we merge it into master it will be visible here https://plugins.jenkins.io/lockable-resources assap. But next release will be published later.  This might lead to misunderstandings.

Reply:
There are a few alternatives:

* Hide the documentation changes temporarily as comments, then uncomment them immediately before the release that delivers the feature.
* Remove the documentation changes and create a separate pull request to be merged just before the release with the functionality
* Temporarily note in the documentation that the feature is not yet available in a release and rely on users to be understanding that the documentation is ahead of the release
* Change the documentation URL in the plugin to use a version specific copy, then release a new version of the plugin with the updated documentation URL.  That has the positive result of only showing documentation changes that are specific to a release.  It has the negative result that all documentation corrections are hidden until the next release.  I dislike that option personally because I find that most of my documentation changes are release independent improvements, not unreleased feature descriptions